src/libra_py/workflows/nbra/lz.py

```diff
@@ -228,6 +228,117 @@ def Belyaev_Lebedev(Hvib, params):
 
 
 
+
+def adjust_SD_probabilities(P, params):
+    """
+    Adjusts the surface hopping probabilties computed according to Belyaev-Lebedev's work by setting the probability
+    to hop between Slater determinants that differ by more than 1 electron to zero.  
+
+    Args:
+        P ( list of lists of MATRIX ): each element of P contains the probability matrix according to the NBRA BLSH method
+        params ( dictionary ): control parameters
+
+        * **params["excitations"]** ( list of lists of lists ) : SDs themselves (QE or dftb+ non-tddftb) or the SD transitions (tddftb). If the excitations are non-changing, then the

```

Comment:
please explain what would params["excitation"][i][j][k] mean? essentially, what each dimension is?